src/lib/components/comparisonTable/hooks/useComparisonTable.ts

```diff
@@ -56,12 +56,15 @@ export const useComparisonTable = () => {
       return;
     }
 
+    const headerWidth = headerRef.current.getBoundingClientRect().width;
+
     const currentTabIndex = Math.round(
-      headerRef.current.scrollLeft /
-        headerRef.current.getBoundingClientRect().width
+      headerRef.current.scrollLeft / headerWidth
     );
 
-    setSelectedTabIndex(currentTabIndex);
+    if (headerWidth < 544) {
```

Comment:
544 is the mobile breakpoint where the table collapses to a single column table.